.github/workflows/contributors.yml

```diff
@@ -7,8 +7,8 @@ jobs:
   contributors:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: wow-actions/contributors-list@v1
+      - uses: actions/checkout@v4
+      - uses: wow-actions/contributors-list@a9cb338523e20921d1a535d29f00dc8458fa8fe2
```

Comment:
What commit does this correspond to? I don't see it in https://github.com/wow-actions/contributors-list so I'm confused
Also wondering why this is changed, the contributors list is working currently

Reply:
I don't remember why I changed this back then, I just brought in the change via cherry-picking. I think it's a valid point.